assets/manifest-firefox.json

```diff
@@ -0,0 +1,39 @@
+{
+  "manifest_version": 3,
+
+  "name": "K2 for GitHub",
+  "version": "1.3.66",
```

Comment:
I believe there are instructions in the README to update this version for every PR. Can you please update the instructions to include this manifest file too?

Reply:
done